core/templates/pages/exploration-editor-page/services/user-exploration-permissions.service.ts

```diff
@@ -35,13 +35,21 @@ export class UserExplorationPermissionsService {
 
   static permissionsPromise: Promise<ExplorationPermissions> = null;
 
-  async getPermissionsAsync(): Promise<ExplorationPermissions> {
+  async getPermissionsAsync():
+  Promise<ExplorationPermissions> {
```

Comment:
Why this change? (Please do a self-review and add a comment for reviewers for unexpected changes.)

Reply:
Sorry unexpected change

Reply:
Fixed